### DISCUSSION TITLE
Issues With Giving Probabilities On Daily Games

### DISCUSSION THREAD

**Discussion by mitchellz4** [mitchellz4/MLB-Model#2] [Ideas]
Today was the first day testing the model on daily games. It is appearing like the model is giving about the same probability of winning to each game. I think that there is a problem in the model but unsure of what it could be. Maybe overfitting?

**Comment by aaaclones** [comment]
i just uploaded the files i wont be able to run it until later ... have you tried eliminating some of the columns or run some sort of sckit ?

**Comment by mitchellz4** [comment]
No I have not, this could be something to explore though.

**Comment by aaaclones** [comment]
we wont be able to determine overfitting before we can parse the data , we dont want to be underfitting as well ..

**Comment by mitchellz4** [comment]
Okay, this is my first machine learning “project” so I’ve just been trying
to learn as I go. I might have misunderstood your question then. What do
you mean by removing columns ?

**Reply by aaaclones** [reply]
if you can see how Kyle did his Nba he's got columns of stats .. he picked and chose those key data points to run , if you account for the stats each player/team has it will most certainly be overfitting , so you will have to choose the columns you want to have and remove to test the model .. i can say you can see how it does within a few days or so , if the model is good you can easily determine if you chose the right data sets or data points in your columns

**Comment by sportsfan69** [comment]
I finally got the model to load and work correctly. I am seeing the same issue @mitchellz in which all the predictions are given the same percentage. Any one have any suggestions on how to modify the columns on which are important for creating a predictive model?

**Reply by mitchellz4** [reply]
Yeah, I am unsure for the reasoning of this. I am going to make this repository public in hopes that someone has an idea!

**Reply by mitchellz4** [reply]
I think the reason for this could have been overfitting with the data but with testing and tweaking to make sure it does not overfit the probabilities still seem to be the same. I think I have an idea to change the data the model is working off of.  I do not have much time to explore this theory right now. just a thought.
